include/SDL3/SDL_gpu.h

```diff
@@ -2262,9 +2264,15 @@ extern SDL_DECLSPEC SDL_GPUComputePipeline *SDLCALL SDL_CreateGPUComputePipeline
     SDL_GPUDevice *device,
     const SDL_GPUComputePipelineCreateInfo *createinfo);
 
+#define SDL_PROP_GPU_CREATECOMPUTEPIPELINE_NAME_STRING "SDL.gpu.createcomputepipeline.name"
```

Comment:
The naming convention is SDL.namespace.object.create.property, e.g.
```suggestion
#define SDL_PROP_GPU_COMPUTEPIPELINE_CREATE_NAME_STRING "SDL.gpu.computepipeline.create.name"
```

Reply:
Looks like SDL_PROP_GPU_CREATETEXTURE_D3D12_CLEAR_R_FLOAT etc don't follow this, which is unfortunate. Should we alias those?

Reply:
Just fix them. We haven't hit 3.2.0 release yet, I think this is worth fixing and low risk of breaking existing apps

Reply:
That can come in a separate commit though to keep things clean.